src/sumOfSquares.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const sum = require('./sum');
+const square = require('./square');
```

Comment:
will be better to get needed functions from the modules via destructuring
```suggestion
const { sum } = require('./sum');
const { square } = require('./square');
```